lib/administrate/field/has_many.rb

```diff
@@ -44,6 +46,10 @@ def more_than_limit?
         data.count(:all) > limit
       end
 
+      def data
+        @data ||= associated_class.none 
+      end
```

Comment:
Is this the best way to implement the null object pattern for the `data` method?